.github/scripts/install-torch-tensorrt.sh

```diff
@@ -8,6 +8,11 @@ PLATFORM=$(python -c "import sys; print(sys.platform)")
 pip install --pre ${TORCH_TORCHVISION} --index-url ${INDEX_URL}
 pip install --pre -r ${PWD}/tests/py/requirements.txt --use-deprecated legacy-resolver
 
+export CUDA_HOME="$(echo ${CUDA_PATH} | sed -e 's#\\#\/#g')"
+export TORCH_INSTALL_PATH="$(python -c "import torch, os; print(os.path.dirname(torch.__file__))" | sed -e 's#\\#\/#g')"
+
+cat ${PWD}/toolchains/ci_workspaces/MODULE.bazel.tmpl | envsubst > ${PWD}/MODULE.bazel
```

Comment:
Was the addition of these three lines necessary? `MODULE.bazel` is only used in build job to create the wheel. But `install-torch-tensorrt.sh` is only used in tests job so `MODULE.bazel` is having no use here.

Reply:
@HolyWu  thanks for pointing out this, it is indeed not required, removed as suggested.